src/twisted/internet/defer.py

```diff
@@ -1629,7 +1757,7 @@ def returnValue(val: object) -> NoReturn:
 
 
 @attr.s(auto_attribs=True)
-class _CancellationStatus:
+class _CancellationStatus(Generic[_DeferredResultT]):
```

Comment:
At least `_inlineCallbacks` uses `_CancellationStatus` unparameterized.

Reply:
Looks like that's the only place in the codebase that it's used, so fixed :)